siteapp/settings_application.py

```diff
@@ -1,10 +1,9 @@
-# nothing here
-
 import re
+import sys
 from .settings import *
 
 INSTALLED_APPS += [
-    #'debug_toolbar',
+    'debug_toolbar',
```

Comment:
@davidpofo Do we want Debig toolbar running all the time?

Reply:
Added a new enable_toolbar field in environment.json/settings to turn this on/off.

Reply:
Please remember to `enable_toolbar` param to documentation: https://govready-q.readthedocs.io/en/latest/installation-and-setup/configuration-with-environment-variables.html